packages/config/src/plugins.js

```diff
@@ -53,6 +54,11 @@ module.exports = ({
           },
           ...(replacePlugin || []),
         ]),
+        new DefinePlugin({
+          // we have to wrap the appname string in another string because of how define plugin explodes strings
+          CRC_APP_NAME: JSON.stringify(insights?.appname),
```

Comment:
We might also want to add a new global variable for the `CRC_APP_NAME` variable to the eslint config.